### ISSUE TITLE
Confusing documentation in modelParameters.mdl

### ISSUE THREAD

**Issue by ArtPoon** [veg/hyphy#640]
In the `Model Options` dialog, the entry for `Global w/variation` reads:
```
Model parameters are shared by all branches, branch lengths come from a user-chosen distribution, whose parameters are estimated.
```
Now, I'm fairly certain that this option enables rate variation across *sites*, not branches.  Am I wrong?

**Comment by spond** [comment]
Dear @ArtPoon,

Yes, this text is wrong. Could you please fix and issue a pull request to 2.3.4 branch, which is planned for release at the end of the month.

Thanks!
Sergei

**Comment by ArtPoon** [comment]
will do
